content/en/tracing/trace_collection/trace_context_propagation/_index.md

```diff
@@ -107,6 +111,7 @@ In addition to the environment variable configuration, you can also update the p
 [4]: https://github.com/openzipkin/b3-propagation#multiple-headers
 [5]: https://docs.aws.amazon.com/xray/latest/devguide/xray-concepts.html#xray-concepts-tracingheader
 [6]: #none-format
+[7]: https://www.w3.org/TR/baggage/
```

Comment:
```suggestion
```

Reply:
This link will need to move outside of the tab structure to line `651` in order to work